stdlib/tempfile.pyi

```diff
@@ -546,6 +546,7 @@ class SpooledTemporaryFile(IO[AnyStr], _SpooledTemporaryFileBase):
     def seek(self, offset: int, whence: int = ...) -> int: ...
     def tell(self) -> int: ...
     def truncate(self, size: int | None = None) -> None: ...  # type: ignore[override]
+    _file: IO[AnyStr]
```

Comment:
One more small nit: I would prefer this to be in the beginning of the class, similarly to the `name` attribute in `TemporaryDirectory` below.